src/app/routes/utils/fetchPageData/index.js

```diff
@@ -35,7 +35,7 @@ export const getUrl = pathname => {
 export default async ({ path, pageType }) => {
   const url = getUrl(path);
 
-  logger.info(DATA_REQUEST_RECEIVED, { url, pageType });
+  logger.info(DATA_REQUEST_RECEIVED, { data: url, pageType, path });
```

Comment:
The reason for the logging changes in this file is to make sure that logs are capturing the same data as will be captured in CloudWatch (path will be `/pidgin/12345678` instead of `https://www.bbc.com/pidgin/12345678.json`